NOTICE

```diff
@@ -24,43 +45,31 @@ The derived files are:
 - baremaps-core/src/main/proto/vector_tile.proto
 
 This product includes code derived from Palantir Streams.
-Apache License, Version 2.0
+Apache License 2.0.
 Please visit the following URL for the full text of the Palantir license:
 https://github.com/palantir/streams/blob/develop/LICENSE
 The derived files are:
 - baremaps-core/src/main/java/org/apache/baremaps/stream/BufferedSpliterator.java
 
 This product includes code derived from Planetiler.
-Apache License, Version 2.0
+Apache License 2.0.
 Please visit the following URL for the full text of the Planetiler license:
 https://github.com/onthegomap/planetiler/blob/main/LICENSE
 The derived files are:
 - baremaps-core/src/main/java/org/apache/baremaps/database/collection/MemoryAlignedDataMap.java
 - baremaps-core/src/main/java/org/apache/baremaps/database/collection/MonotonicDataMap.java
 - baremaps-core/src/main/java/org/apache/baremaps/database/collection/MonotonicFixedSizeDataMap.java
 
+This product includes test data copied from PMTiles.
+BSD-3-Clause license.
+Please visit the following URL for the full text of the PMTiles license:
+https://github.com/protomaps/PMTiles
+The copied files are located in:
+- baremaps-core/src/test/resources/pmtiles/
+
 This product includes code derived from pyosmium.
 BSD 2-Clause "Simplified" License.
 Please visit the following URL for the full text of the pyosmium license:
 https://github.com/osmcode/pyosmium/blob/master/LICENSE.TXT
 The derived files are:
 - baremaps-core/src/main/java/org/apache/baremaps/openstreetmap/state/StateReader.java
```

Comment:
https://github.com/apache/incubator-baremaps/blob/main/baremaps-core/src/main/java/org/apache/baremaps/openstreetmap/state/StateReader.java#L40-L43

In general, we need to preserve the Pyosmium license header. If we modify it, we can also add the Apache license header.

Here is a good example.  https://issues.apache.org/jira/browse/LEGAL-626